src/unistd.rs

```diff
@@ -3355,6 +3355,55 @@ mod getres {
     }
 }
 
+feature! {
+#![feature = "process"]
+#[cfg(target_os = "freebsd")]
+pub mod rfork_freebsd {
+    use crate::errno::Errno;
+    use crate::Result;
+    use super::ForkResult;
+    use super::Pid;
+
+    libc_bitflags! {
+        /// Flags for [`rfork`]
+        /// subset of flags supported by FreeBSD 12.x and onwards
+        /// with a safe outcome, thus as `RFMEM` can possibly lead to undefined behavior,
+        /// it is not in the list. And `rfork_thread` is deprecated.
+        pub struct RforkFlags: libc::c_int {
```

Comment:
Seems like there is a `RFSIGSHARE` flag that is currently missing, though it is also not available in `libc`

Reply:
Noticed this PR https://github.com/rust-lang/libc/pull/3457, thanks for doing it!